assets/src/dashboard/theme.js

```diff
@@ -272,6 +272,8 @@ const theme = {
     small: {
       desktop: 20,
       min: 10,
+      largeDisplayPhone: 10,
```

Comment:
thanks for doing this, the only other spot this is needed until i can clean up the gutters to be less confusing is line 287 (`pageHorizontalGutter.largeDisplayPhone`) to 10 to correspond since the gutter.small and gutter.large are not easily accessible when in the usePageSize all i have is a breakpoint, no knowledge of small or large.